ovos_number_parser/__init__.py

```
@@ -1,5 +1,6 @@
 from typing import Union
 
+from ovos_number_parser.util import Scale, GrammaticalGender
```

Ruff (0.12.2)

`ovos_number_parser.util.GrammaticalGender` imported but unused; consider removing, adding to `__all__`, or using a redundant alias

(F401)